experimental/middleware/middleware.go

```diff
@@ -4,11 +4,33 @@
 package experimental
 
 import (
+	"embed"
 	"net/http"
 
 	"github.com/corazawaf/coraza/v3"
 )
 
+//go:embed error_template.html error_template.html
+var embededTemplates embed.FS
```

Comment:
So... are these going to be `html/template`s or just html code? At first read I thought they were not static html files only.

In summary:
- are these meant to be static html?
- wouldn't be more interesting to use `html/template` templates that can be filled with details about the transaction? E.g. "Some error found. Contact your admin with this code: { < the transaction ID> }"`

If we use static html, maybe removing the "_template" in the name will make more sense, and use "default_error.html" and "default_interruption.html" instead?